custom/nic_compliance/static/nic_compliance/js/encoder.js

```diff
@@ -62,7 +62,7 @@ hqDefine('nic_compliance/js/encoder', [
                     $field.val(passwordEncoder.encode(unencodedValue));
                 });
                 $(document).on("ajaxComplete", function (e, xhr, options) {
-                    if ($form.attr("action").endsWith(options.url)) {
+                    if ($form.attr("action") && $form.attr("action").endsWith(options.url)) {
```

Comment:
Curious why this changed? It's reasonable code, I just mean that if this check wasn't necessary previously and is now, it seems like this code might be executing under some set of conditions where it wasn't previously.

Reply:
it was throwing an error on the change my password page